client/state/email-accounts/test/reducer.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { usersReducer } from '../reducer';
+
+import { EMAIL_ACCOUNTS_REQUEST_SUCCESS } from 'state/action-types';
+
+describe( "emailAccountsReducer's", () => {
+	const account = {
+		mailbox: 'a',
+		domain: 'test.blog',
+		email: 'a@test.blog',
+		firstname: 'User',
+		lastname: 'One',
+		fullname: 'User One',
+		site_id: 1,
+		suspended: false,
```

Comment:
This should be `is_suspended`. Interestingly the reducer didn't pick the issue despite the having the property listed in the schema.